### PR TITLE
VEN-815 | Mail notification templates

### PR DESCRIPTION
**Pull request by a-rmz**
## Description :sparkles:
* Adapt the `Order approved` notification type for the email template
* Add the `Berth/WS Application approved` notification type (will be used in the future)
* Add a `DummyContext` for the `Order approved` notifications
* Fix a weak query for `ContentType` that was making the tests break

### Closes :no_good_woman:
**[VEN-815](https://helsinkisolutionoffice.atlassian.net/browse/VEN-815):** Insert email templates into the database

### Related :handshake:
#274: VEN-547 | Add a mutation to approve orders and send emails

### Automated tests :gear:️
```shell
$ pytest
```

## Additional notes :spiral_notepad:
This changes are required to send the invoice emails, the notification templates will later be added to the database.

### CODE REVIEW

File: payments/models.py [hunk 1]
```diff
@@ -659,12 +672,16 @@ def pretax_price(self):
...
     def __str__(self):
         return (
-            f"{self.product.name} - {self.product.price_value}"
-            f"{'€' if self.product.price_unit == PriceUnits.AMOUNT else '%'}"
+            (
+                f"{self.product.name} - {self.product.price_value}"
+                f"{'€' if self.product.price_unit == PriceUnits.AMOUNT else '%'}"
+            )
+            if self.product
+            else f"{_('No product')} - {self.price}€"
```
Comment (igordavydsson) [inline payments/models.py:684]: these are very welcome, stumbled into them yesterday 😅

File: applications/notifications.py [hunk 2]
```diff
@@ -26,6 +26,14 @@ class NotificationType(TextChoices):
...
         _("Unmarked winter storage application created"),
     )
+    BERTH_APPLICATION_CONFIRMED = (
+        "berth_application_confirmed",
+        _("Berth application confirmed"),
+    )
+    WINTER_STORAGE_APPLICATION_CONFIRMED = (
+        "winter_storage_application_confirmed",
+        _("Winter storage application confirmed"),
+    )
```
Comment (igordavydsson) [inline applications/notifications.py:36]: so, what are we gonna use these for?

Comment (a-rmz) [reply applications/notifications.py:36]: Not 100% sure when, but we have templates for `Application confirmation`, so just preventing that. HOWEVER, now that I think about it, they could (and maybe should) be left out of this PR.

Comment (igordavydsson) [reply applications/notifications.py:36]: if we are not really using them anywhere, I would leave them out for now

Comment (igordavydsson) [reply applications/notifications.py]: I am in general not sure what these are supposd to be for 🤔  I was thinking that `ORDER_APPROVED` already is `APPLICATION_HANDLED` notification as such

Comment (a-rmz) [reply applications/notifications.py]: Yeah, I'm not sure either, I was just preparing fot the templates that Tatu made :derp:

Comment (santtul) [reply applications/notifications.py]: voting for leaving it out for now

Comment (igordavydsson) [reply applications/notifications.py]: if you mean these
image
then they are `BERTH_APPLICATION_CREATED`, `WINTER_STORAGE_APPLICATION_CREATED` and `UNMARKED_WINTER_STORAGE_APPLICATION_CREATED` effectively, but with new looks

File: applications/notifications.py [hunk 3]
```diff
@@ -63,5 +63,13 @@ class NotificationType(TextChoices):
...
             "area_choices": sorted(winter_area_choices, key=lambda c: c.priority),
         },
+        NotificationType.BERTH_APPLICATION_CONFIRMED: {
+            "application": berth_application,
+            "harbor_choices": sorted(harbor_choices, key=lambda c: c.priority),
+        },
+        NotificationType.WINTER_STORAGE_APPLICATION_CONFIRMED: {
+            "application": winter_storage_application,
+            "area_choices": sorted(winter_area_choices, key=lambda c: c.priority),
+        },
```
Comment (igordavydsson) [inline applications/notifications.py:73]: 👀

File: applications/notifications.py (hunk 2 above)
Comment (TatuArvela) [reply applications/notifications.py]: `application-confirmation-##.html` are all old templates updated to HDS. They might be a red herring in this discussion, check with Kalle

File: applications/notifications.py (hunk 3 above)
Comment (a-rmz) [reply applications/notifications.py]: 🙅

File: applications/notifications.py (hunk 2 above)
Comment (a-rmz) [reply applications/notifications.py]: AHA! Alright, then definitely no need for them 👍

**Review by santtul — APPROVED** [review]
👍

**Pull request merged**